console/console-init/ui/src/modules/address/components/AddressConfiguration/AddressConfiguration.tsx

```diff
@@ -144,6 +156,51 @@ const AddressConfiguration: React.FunctionComponent<IAddressConfigurationProps>
                 />
               </FormGroup>
             )}
+            {((type && type.toLowerCase() === "subscription") ||
+              (type && type.toLowerCase() === "queue")) && (
+              <FormGroup
+                label="Deadletter Queue"
```

Comment:
Please use the term Deadletter Address and Expiry Address rather than Deadletter Queue and Expiry Queue.
```suggestion
                label="Deadletter Address"
```

Reply:
Also I noticed the "Created" field is missing from the header for addresses.